### PR TITLE
contrib/pgn-extract: new package (22.11)

### PR DESCRIPTION
**Pull request by shtayerc**

### CODE REVIEW

File: contrib/pgn-extract/template.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pkgname = "pgn-extract"
+pkgver = "22.11"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_check_target = "all"
```
Comment (nekopsykose) [inline contrib/pgn-extract/template.py:6]: pass make_build_args = ["OPTIMISE="] too  

also add `$(CFLAGS)` to

```makefile
pgn-extract : $(OBJS)
	$(CC) $(DEBUGINFO) $(ORIGCFLAGS) $(CPPFLAGS) $(LDFLAGS) $(OBJS) $(LIBS) -o pgn-extract
```

File: contrib/pgn-extract/template.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+make_check_args = ["--directory", "test"]
+make_use_env = True
+hostmakedepends = ["gmake"]
+pkgdesc = "Portable Game Notation (PGN) Manipulator for Chess Games"
+maintainer = "shtayerc <david.murko@mailbox.org>"
+license = "GPL-3.0-or-later"
+url = "https://www.cs.kent.ac.uk/people/staff/djb/pgn-extract"
+source = f"{url}/pgn-extract-{pkgver.replace('.' , '-')}.tgz"
```
Comment (nekopsykose) [inline contrib/pgn-extract/template.py:14]:
```suggestion
source = f"{url}/pgn-extract-{pkgver.replace('.', '-')}.tgz"
```

File: contrib/pgn-extract/template.py (hunk 1 above)
Comment (shtayerc) [reply contrib/pgn-extract/template.py:7]: If i just patch $(CFLAGS) here it breaks. I guess we only need cflags from cports

Comment (nekopsykose) [reply contrib/pgn-extract/template.py:7]: ah, sure, cause `-c` is in them

**Pull request merged**